src/Cradle/ProcessConfiguration/Helpers.hs

```diff
@@ -38,6 +39,14 @@ setDelegateCtrlC :: ProcessConfiguration -> ProcessConfiguration
 setDelegateCtrlC config =
   config {delegateCtlc = True}
 
+setEnviron ::
+  ConvertibleStrings s String =>
+  Maybe [(s, s)] ->
```

Comment:
```suggestion
  [(s, s)] ->
```

Why would a user ever want to set this to `Nothing`? I mean I can *imagine* some use-cases, but they seem pretty contrived. Do you have a good use-case for that?